tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
```

Comment:
Could you add black, mypy and bandit to tox, as we are already using typing?

Reply:
I’ve just checked it, ruff can replace black and bandit, so I’ll configure it this way. Have you checked if mypy is happy with the code base?

Reply:
It wasn’t, now it is. 😀